### ISSUE TITLE
Info - CATcher : Unable to confirm or deny if reply "wrong phase" is correct

### ISSUE THREAD

**Issue by 9fc70c892** [nus-cs2103-AY2223S2/forum#391]
Just a heads up. 

There is no way to confirm from CATcher that what the developers claim to be in the wrong phase, is actually in wrong phase. You will need to look it up on Github to confirm if that is true.

TLDR:
I do not recommend disputing "wrong phase" replies it unless you can prove it otherwise.

**Comment by damithc** [comment]
@9fc70c892 Did you mean functionality or feature flaw bugs reported during phase 1 part II?

**Comment by 9fc70c892** [comment]
> @9fc70c892 Did you mean functionality or feature flaw bugs reported during phase 1 part II?

Yes

**Comment by damithc** [comment]
@9fc70c892 Yes, currently CATcher does not show the timestamp of bug reports.  For now, you can refer to the issue in GitHub to find the time a bug was reported, if you want to verify if it was indeed reported during Phase 1 Part II (which was 3.10pm-3.40pm)
